utility/rtos_compatibility_layers/FreeRTOS/tx_freertos.c

```diff
@@ -33,6 +33,10 @@
 /*                                            start flag, corrected stack */
 /*                                            allocation size,            */
 /*                                            resulting in version 6.1.12 */
+/*  xx-xx-xxxx     Xiuwen Cai               Modified comment(s), and      */
```

Comment:
Could you please update the release date (12-31-2023) and version number (6.4.0)?

Reply:
Updated.